README.md

```diff
@@ -4,6 +4,14 @@ This repo hosts the Dockerfiles used to generate the official images for OSRF pr
 ##### Dockerfile CI [![Build Status](https://travis-ci.org/osrf/docker_images.svg?branch=master)](https://travis-ci.org/osrf/docker_images)
 
 ## Official Library
+### [ROS2 ![Docker Pulls](https://img.shields.io/docker/pulls/_/ros2.svg) ![Docker Stars](https://img.shields.io/docker/stars/_/ros2.svg)](https://registry.hub.docker.com/_/ros2/)
```

Comment:
It looks like ros2 doesn't exist on the official docker hub. Is the goal here to make the official images match the split in `osrf/ros` and `osrf/ros2`?

Reply:
I've now reworked the PRs to consolidate the versions into just the one official ros repo.